scripts/pi-hole/js/queries.js

```diff
@@ -78,7 +78,7 @@ $(function () {
       var replyid = parseInt(data[5], 10);
       // DNSSEC status
       var dnssecStatus;
-      var ede = data[11];
+      var ede = (data[11]) ? data[11] : '';
```

Comment:
```suggestion
      var ede = data[11] ? data[11] : '';
```

for the CI test to pass.